docs/user-guide/prereqs.md

```diff
@@ -13,6 +13,15 @@ If you downloaded a pre-compiled [release](https://github.com/kubernetes/kuberne
 
 If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
 
+If you don't have the source code checked out you can download the compiled kubectl binary for each release with curl
+
+```shell
+# Linux
+curl -LO https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl
```

Comment:
We should give an example of `${K8S_VERSION}` or show its format, such as `v1.4.6`